test/e2e/basic_test.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/apis/sparkoperator.k8s.io/v1alpha1"
```

Comment:
The order of import paths needs to be fixed.

Reply:
Hmm aren't they already ordered?  When I made any changes, my editor (GoLand) told me that my imports were not sorted.

Reply:
All other source files use the following order: built-in packages, github.com packages, k8s.io packages, and finally the operator packages. Just want to be consistent with the other files.

Reply:
Sure. Done.